src/chi/themes/centurylink/index.scss

```
@@ -0,0 +1,93 @@
+@import '../../_global-variables';
+@import '_variables';
+@import '../../_global-mixins';
+@import '_mixins';
+
+// sass-lint:disable clean-import-paths
+@import '../../foundations/fonts/fonts';
+@import '../../components/icons/webfont-font-face';
+@import '../../foundations/reset/reset';
+@import 'typography/root-typography';
+
+$theme: 'ctl';
```

for consistency with theme folder name

```suggestion
$theme: 'centurylink';
```